diff -ur open-vm-tools-9.4.6-1770165/lib/include/vm_basic_types.h open-vm-tools-9.4.6-1770165-gcc5/lib/include/vm_basic_types.h
--- open-vm-tools-9.4.6-1770165/lib/include/vm_basic_types.h	2014-07-02 00:21:14.000000000 +0200
+++ open-vm-tools-9.4.6-1770165-gcc5/lib/include/vm_basic_types.h	2015-11-22 00:35:43.844044867 +0100
@@ -83,7 +83,7 @@
 
 /* STRICT ANSI means the Xserver build and X defines Bool differently. */
 #if !defined(_XTYPEDEF_BOOL) && \
-    (!defined(__STRICT_ANSI__) || defined(__FreeBSD__) || defined(__MINGW32__))
+    (!defined(__STRICT_ANSI__) || defined(__FreeBSD__) || defined(__MINGW32__) || !defined(Bool))
 #define _XTYPEDEF_BOOL
 typedef char           Bool;
 #endif
diff -ur open-vm-tools-9.4.6-1770165/lib/include/vmblock.h open-vm-tools-9.4.6-1770165-gcc5/lib/include/vmblock.h
--- open-vm-tools-9.4.6-1770165/lib/include/vmblock.h	2014-07-02 00:21:14.000000000 +0200
+++ open-vm-tools-9.4.6-1770165-gcc5/lib/include/vmblock.h	2015-11-22 00:37:51.554524645 +0100
@@ -148,7 +148,7 @@
 # define VMBLOCK_DEVICE_MODE           VMBLOCK_FUSE_DEVICE_MODE
 # define VMBLOCK_MOUNT_POINT           VMBLOCK_FUSE_MOUNT_POINT
 
-#elif defined(linux)
+#elif defined(__linux__)
 # define VMBLOCK_ADD_FILEBLOCK         98
 # define VMBLOCK_DEL_FILEBLOCK         99
 # ifdef VMX86_DEVEL
